packages/blockchain-wallet-v4-frontend/src/data/components/send/actionTypes.ts

```diff
@@ -6,3 +6,6 @@ export const FETCH_PAYMENTS_ACCOUNT_EXCHANGE_FAILURE =
   '@EVENT.SEND.FETCH_PAYMENTS_ACCOUNT_EXCHANGE_FAILURE'
 export const FETCH_PAYMENTS_ACCOUNT_EXCHANGE_SUCCESS =
   '@EVENT.SEND.FETCH_PAYMENTS_ACCOUNT_EXCHANGE_SUCCESS'
+
+export const FETCH_PAYMENTS_TRADING_AACCOUNTS_SUCCESS =
```

Comment:
small misspelling

Reply:
Any reason to only have SUCCESS actionType?

Usually in this scenario we would have 4:
1. actionType to kick off a saga
2. SUCCESS
3. FAILURE
4. LOADING

Reply:
can add others of course, didn't sow benefit to use them but I'll follow up standards

Reply:
How do you handle the case of the the request failing for instance?